internal/apiresource/service.go

```diff
@@ -369,13 +371,23 @@ func (d *Service) ingressToService(ingress networkingv1.Ingress) []runtime.Objec
 	return objs
 }
 
+func (d *Service) createRoutes(service irtypes.Service, ir irtypes.IR) [](*okdroutev1.Route) {
```

Comment:
Should we also fix ingressToRoute function and servicetoroute function? Also the routetoingress and other similar functions?